src/components/MultiSelect/placeholderOption.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import StyledHeader from '../Option/styled/header';
+import StyledHeaderLabel from '../Option/styled/headerLabel';
+import { StyledPrimitiveCheckbox } from './styled';
+
+function PlaceholderOption(props) {
+    const { label, title, isChecked, onClick, ...rest } = props;
+
+    return (
+        <StyledHeader title={title} role="presentation" onMouseDown={onClick}>
+            <StyledPrimitiveCheckbox type="checkbox" label="" checked={isChecked} {...rest} />
+            <StyledHeaderLabel>{label}</StyledHeaderLabel>
```

Comment:
I see there are two like this, other in the InternalDropdown, both are needed?

Reply:
Nop, just a leftover when moving the 'Select all' logic to the InternalDropdown component. Done.